main/curl/APKBUILD

```diff
@@ -90,7 +90,7 @@ build() {
 		--enable-unix-sockets \
 		--without-libidn \
 		--without-libidn2 \
-		--with-libssh2 \
+		--without-libssh2 \
```

Comment:
It would probably be a good idea to add a link the redmine bug in this line to ensure that this isn't enabled again in the near future.

Reply:
Applied, thanks for the suggestion